From 6818e4872af491d7de4e364f183e7e24d8665799 Mon Sep 17 00:00:00 2001
Message-ID: <6818e4872af491d7de4e364f183e7e24d8665799.1771339327.git.sergeyb@tarantool.org>
In-Reply-To: <cover.1771339327.git.sergeyb@tarantool.org>
References: <cover.1771339327.git.sergeyb@tarantool.org>
From: Sergey Bronnikov <sergeyb@tarantool.org>
Date: Tue, 17 Feb 2026 17:41:33 +0300
Subject: [PATCH luajit 3/3] cmake: bump libiconv with tag
To: tarantool-patches@dev.tarantool.org, Sergey Kaplun <skaplun@tarantool.org>, Maxim Kokryashkin <m.kokryashkin@tarantool.org>

NO_CHANGELOG=internal
NO_DOC=internal
NO_TEST=internal
NO_DEPS_UPDATE
---
 cmake/BuildICONV.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/BuildICONV.cmake b/cmake/BuildICONV.cmake
index 64df3b11d4..fced50beb2 100644
--- a/cmake/BuildICONV.cmake
+++ b/cmake/BuildICONV.cmake
@@ -1,4 +1,4 @@
-set(ICONV_VERSION 1.19)
+set(ICONV_VERSION 1.20)
 set(ICONV_HASH d718cd5a59438be666d1575855be72c3)
 set(ICONV_INSTALL_DIR ${BUNDLED_LIBS_INSTALL_DIR}/iconv-prefix)
 set(ICONV_INCLUDE_DIR ${ICONV_INSTALL_DIR}/include)
--
2.43.0
